koin-projects/koin-core/src/main/kotlin/org/koin/core/KoinApplication.kt

```diff
@@ -83,16 +83,19 @@ class KoinApplication private constructor() {
 
     /**
      * Set Koin Logger
-     * @param level
      * @param logger - logger
      */
-    @JvmOverloads
-    fun logger(level: Level = Level.INFO, logger: Logger = PrintLogger()): KoinApplication {
+    fun logger(logger: Logger): KoinApplication {
         KoinApplication.logger = logger
-        KoinApplication.logger.level = level
         return this
     }
 
+    /**
+     * Set Koin to use [PrintLogger], by default at [Level.INFO]
+     */
+    @JvmOverloads
+    fun printLogger(level: Level = Level.INFO) = this.logger(PrintLogger(level))
```

Comment:
This is the primary DSL change

Reply:
Ok I see the problem with my first proposed DSL value